lib/platform/github/index.js

```diff
@@ -688,7 +689,7 @@ async function addReviewers(prNo, reviewers) {
 async function addLabels(issueNo, labels) {
   logger.debug(`Adding labels ${labels} to #${issueNo}`);
   const repository = config.parentRepo || config.repository;
-  if (Array.isArray(labels) && labels.length) {
+  if (is.array(labels) && labels.length) {
```

Comment:
Replace with `!is.empty(labels)`

Reply:
this one caused a snapshot test to fail unfortunately, so for the time being i've left it as is